### ISSUE TITLE
WideAndDeepExample does not accept Float continuousCols

### ISSUE THREAD

**Issue by bethunebtj** [intel-analytics/analytics-zoo#685]
The code in WideAndDeepExample.scala seems that does not support multi value indicators , like this：

```
  //com.intel.analytics.zoo.models.recommendation.Utils.scala
  // setup deep tensor
  def getDeepTensor(r: Row, columnInfo: ColumnFeatureInfo): Tensor[Float] = {
    val deepColumns1 = columnInfo.indicatorCols
    val deepColumns2 = columnInfo.embedCols ++ columnInfo.continuousCols
    val deepLength = columnInfo.indicatorDims.sum + deepColumns2.length
    val deepTensor = Tensor[Float](deepLength).fill(0)

    // setup indicators
    var acc = 0
    (0 to deepColumns1.length - 1).map {
      i =>
        val index = r.getAs[Int](columnInfo.indicatorCols(i))
        val accIndex = if (i == 0) index
        else {
          acc = acc + columnInfo.indicatorDims(i - 1)
          acc + index
        }
        deepTensor.setValue(accIndex + 1, 1)
    }

    // setup embedding and continuous
    (0 to deepColumns2.length - 1).map {
      i =>
        deepTensor.setValue(i + 1 + columnInfo.indicatorDims.sum,
          r.getAs[Int](deepColumns2(i)).toFloat)
    }
    deepTensor
  }
```

The example calls the `Utils.row2Sample()` method， and the method deals with indicators & continuous Cols like the above code.

We can see that it assumes that continuousCols are **integers**  rather than **floats** , and it only takes the first value of indicators. I understand that it's only a demo , but will it be better if we fix this?

**Comment by jason-dai** [comment]
@songhappy please take a look

**Comment by songhappy** [comment]
fixed in #2496 and close this issue
